.version

```diff
@@ -1 +1 @@
-v1.6.0-31d162e*v1.6*1*6*0*0*31d162e
```

Comment:
This shouldn't be required since the version is auto-incremented by [CMakeLists.txt.versioning](https://github.com/awslabs/aws-iot-device-client/blob/main/CMakeLists.txt.versioning)